src/Illuminate/Notifications/ChannelManager.php

```diff
@@ -32,10 +32,10 @@ class ChannelManager extends Manager implements DispatcherContract, FactoryContr
     public function send($notifiables, $notification)
     {
         if ($notification instanceof ShouldQueue) {
-            return $this->queueNotification($notifiables, $notification);
+            $this->queueNotification($notifiables, $notification);
```

Comment:
I added the return back to the queueNotification method, seems I missed the return part of dispatch.